config/app/manifest/de_DE.json

```diff
@@ -0,0 +1,59 @@
+{
+    "name": "App Payment Template",
+    "provider": "Spryker",
+    "description": "App Payment Template",
+    "descriptionShort": "App Payment Template",
+    "url": "https://www.spryker.com",
+    "isAvailable": true,
+    "categories": [
+        "PAYMENT"
+    ],
+    "assets": [
+        {
+            "type": "icon",
+            "url": "https:\\/\\/cdn.dribbble.com\\/userupload\\/4049734\\/file\\/original-c4c801b6e18ed232b9c72fbc9fb0c19f.png?resize=400x0"
```

Comment:
I think the characters shouldn't be skipped.

Reply:
yup. sorry, my bad - forgot to remove escaping